.gitignore

```diff
@@ -276,8 +276,12 @@ src/Tools/PhantomJS/tools/phantomjs/ChangeLog
 src/SFA.DAS.AssessorService.Web.Specflow.Tests/Project/**/*.*
 
 # Minified css and js
```

Comment:
Since this is being done for an unusual reason - because there are included minified files in the repository and a blanket ignore is implicated in a build time bundling issue - a better comment would appropriate here